src/components/Header.vue

```diff
@@ -8,10 +8,11 @@
           width="10%"
           height="10%"
         >
-        <Drawer
-          :drawerflg="drawerFlg"
-        />
       </a>
+      <Drawer
+        :drawerflg="drawerFlg"
```

Comment:
```suggestion
        v-if="drawerFlg"
```
ってしておけば、子コンポーネントにフラグ渡す必要ないかな？